src/components/Header/Header.tsx

```diff
@@ -1,30 +1,46 @@
 import "./Header.scss"
-
-import { useCallback, useState } from 'react';
-
 import Logo from "../Logo/Logo";
 import Search from "../Search/Search";
 
+import { useCallback, useState } from 'react';
+import { CreateMovieModal } from "../Modals/CreateMovieModal/CreateMovieModal";
+import { InformationModal } from "../Modals/InformationModal/InformationModal";
+
 function Header() {
     const [searchQuery, setSearchQuery] = useState<string>('Comedy');
+    const [isCreateMovieModelOpened, setIsCreateMovieModelOpened] = useState<boolean>(false);
+    const [isEditMovieInformationModelOpened, setIsCreateMovieInformationModelOpened] = useState<boolean>(false);
```

Comment:
probably it can be shorter?

Reply:
Thank you for your comment, simplified with removing "Information" from name